3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,117 @@
+//Age-ify (A future age calculator)
+
+let yearOfBirth = 1999;
+let yearFuture = 2027;
+
+let age = yearFuture - yearOfBirth;
+console.log(`you will be in ${age} years old in ${yearFuture}`);
+
```

Comment:
👍 Great work on the logic!
🔍 Just a minor upgrade, if you are not going to update the variable, you can consider using `const`